### PR TITLE
MINOR: fix streams tutorial

### PR DESCRIPTION
**Pull request by ocadaruma**
- Fix inconsistency in store name in streams tutorial

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: docs/streams/tutorial.html [hunk 1]
```diff
@@ -490,25 +490,25 @@ <h4><a id="tutorial_code_wordcount" href="#tutorial_code_wordcount">Writing a th
...
+    Processor: KSTREAM-AGGREGATE-0000000003(stores: [counts-store]) --> KTABLE-TOSTREAM-0000000007 <-- KSTREAM-SOURCE-0000000006
     Processor: KTABLE-TOSTREAM-0000000007(stores: []) --> KSTREAM-SINK-0000000008 <-- KSTREAM-AGGREGATE-0000000003
     Sink: KSTREAM-SINK-0000000008(topic: streams-wordcount-output) <-- KTABLE-TOSTREAM-0000000007
 Global Stores:
   none</code></pre>
 
     <p>
         As we can see above, the topology now contains two disconnected sub-topologies.
-        The first sub-topology's sink node <code>KSTREAM-SINK-0000000004</code> will write to a repartition topic <code>Counts-repartition</code>,
+        The first sub-topology's sink node <code>KSTREAM-SINK-0000000004</code> will write to a repartition topic <code>counts-repartition</code>,
```
Comment (showuon) [inline docs/streams/tutorial.html]: Should it be `counts-repartition` -> `counts-store-repartition` ?

**Review by showuon — COMMENTED** [review]
@ocadaruma , thanks for the nice catch! Left a comment.

File: docs/streams/tutorial.html (hunk 1 above)
Comment (ocadaruma) [reply docs/streams/tutorial.html]: Oh that's right. Let me fix.

**Comment by showuon** [comment]
@ocadaruma , could you also file a PR to kafka-site repo to fix the current doc? Thanks.

**Comment by ocadaruma** [comment]
Sure, let me.

**Comment by ocadaruma** [comment]
@showuon Done: https://github.com/apache/kafka-site/pull/414

**Pull request merged**